docs/source/v3-package-spec.rst

```diff
@@ -646,6 +656,14 @@ Natspec: ``natspec``
     `UserDoc <https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format#user-documentation>`_
     and `DevDoc <https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format#developer-documentation>`_  formats.
 
+Source: ``source``
+^^^^^^^^^^^^^^^^^^
+The global source identifier for the source file from which this contract type was generated.
+
+  :Required: No
+  :Type: String
+  :Value: **must** match a unique source ID included in the `Sources Object`_ for this package.
```

Comment:
This `source` field was added to compensate for the change in a contract alias mentioned above. Contract types simply point to their source via this field.